operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -2135,6 +2296,16 @@ func (rc *ReconciliationContext) ReconcileAllRacks() (reconcile.Result, error) {
 		return recResult.Output()
 	}
 
+	if utils.IsPSPEnabled() {
+		if recResult := psp.CheckEMM(rc); recResult.Completed() {
+			return recResult.Output()
+		}
+
+		// if recResult := psp.CheckPVCHealth(rc); recResult.Completed() {
+		// 	return recResult.Output()
+		// }
```

Comment:
Disabled temporarily since we don't support health timestamp here yet and it made testing EMM easier. We have a ticket to add support for the health timestamp, at which point we will re-enable this.